app/javascript/packs/track-errors.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-underscore-dangle */
+
+import { trackError } from '@18f/identity-analytics';
+import type { WindowWithInitialErrors } from './track-errors-prelude';
+
+declare let window: WindowWithInitialErrors;
+
+const handleErrorEvent = (event: ErrorEvent) => trackError(event.error);
```

Comment:
`ErrorEvent` includes some interesting details like `filename` and `lineno` which don't seem to be included on the error object itself, though I couldn't immediately think of a nice way to reconcile this in the API for the otherwise-generic `trackError(error: Error)` interface.

Reply:
To do: We should probably do some basic filtering, e.g.

```suggestion
function handleErrorEvent(event: ErrorEvent) {
  if (new URL(event.filename).host === location.host) {
    trackError(event.error);
  }
}
```